src/components/document/document.tsx

```diff
@@ -124,6 +124,14 @@ const ViewModeButton = ({ onClick, icon, title }: { onClick: () => void, icon: s
   );
 };
 
+const TitleInfo = ({ docTitle, onClick, title }: { docTitle: string, onClick?: () => void, title?: string }) => {
```

Comment:
```suggestion
const TitleInfo = ({ docTitle, onClick }: { docTitle: string, onClick?: () => void }) => {
```
CSS depends on the `id` being `titlebar-title` -- there's no reason to allow clients to change it. Besides, the property name `title` is confusing given that it refers neither to a tooltip nor to the document title.